starlette/authentication.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 import functools
+import inspect
```

Comment:
Remnant from something else?

Reply:
Yes, thanks!
I thought our linting picked those up but I guess not.